samples/oauth_php/VetsApiClaims/app/Http/Controllers/ClaimsController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\OauthSession;
+
+class ClaimsController extends Controller
+{
+    public function index()
+    {
+        if (null !== session('id') and null !== session('expires_at') ) {
+            $oauth_session = OauthSession::find(session('id'));
+            $url = env('VETS_API_URL') . '/services/claims/v1/claims';
+            $curl = curl_init($url);
```

Comment:
May want to consider using Guzzle - it is a cleaner abstraction layer for http requests, and can work even if curl is not available  on the server - it can be loaded into the IOC too https://medium.com/laravel-5-the-right-way/using-guzzlehttp-with-laravel-1dbea1f633da

Reply:
Changes made